### DISCUSSION TITLE
Disable the typo tolerance

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#176] [Feedback & Feature Proposal]
(Sorry if this already exists, I did not find it using the GitHub search bar)

Some users seem to want to disable the typo tolerance. See [this comment](https://github.com/meilisearch/MeiliSearch/issues/1103#issuecomment-901828680).

## Potential solution

This is already implemented in the search engine (milli), we just need to expose it (MeiliSearch).

We could
- provide a setting
- or provide a parameter during the search time

Both of them are technically possible.

**Comment by gmourier** [comment]
Locked in favor of #312
